src/main/java/saber/Saber.java

```diff
@@ -0,0 +1,136 @@
+package saber;
+
+import saber.commands.SaberCommand;
+
+import saber.exceptions.SaberCommandNotFoundException;
+import saber.exceptions.SaberStorageLoadException;
+import saber.exceptions.SaberStorageStoreException;
+
+import saber.parser.SaberParser;
+import saber.storage.TaskStorage;
+import saber.task.Deadline;
+import saber.task.Event;
+import saber.task.Todo;
+
+import saber.tasklist.TaskList;
+import saber.ui.SaberUI;
+
+import java.util.ArrayList;
+
+/***
+ *  A class that represents the Saber application
+ */
```

Comment:
May want to start javadoc comments using two asterisks only, and end javadocs with fullstop. I noticed this issue in several other places too.
```suggestion
/**
 *  A class that represents the Saber application.
 */
```

Reply:
Ok